tools/installers/install_parallel-wavegan.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```

Comment:
The parallel-wavegan repo supports python 3.6+ and pytorch 1.4+.
We do not need to check the version here since espnet assumes python3.7+ and pytorch 1.4+.

Reply:
And `jq` command is used for only recipe, in our usage we may not need it.

Reply:
So we may make it much simpler.
```bash
#!/usr/bin/env bash

set -euo pipefail

if [ $# != 0 ]; then
    echo "Usage: $0"
    exit 1;
fi

rm -rf ParallelWaveGAN

# ParallelWaveGAN  Commit id when making this PR: `commit 5bef5a0610e5e0d6153b601bbf91c78582260c8f`
git clone https://github.com/kan-bayashi/ParallelWaveGAN.git
cd ParallelWaveGAN
pip install -e .
cd ..
```

Or further simplify like `pip install parallel_wavgan==0.5.4`